src/components/DetailsMenu/index.tsx

```diff
@@ -29,6 +30,11 @@ const DetailsMenu: React.FC<DetailsMenuProps> = ({ "data-cy": dataCy }) => (
       </Column>
     </Row>
     <ButtonRow />
+    <Row>
+      <div>
+        <CLIInstructions />
+      </div>
+    </Row>
   </DetailsMenuCard>
```

Comment:
(comment placement is random) Do you want to update the JIRA button text as shown in the designs? We had talked about changing the copy during design sync because the button text makes it sound like it's a link to JIRA

Reply:
<img width="762" alt="image" src="https://user-images.githubusercontent.com/4605522/232878086-c9a342b5-3019-4add-9f3f-97396d8b0c70.png">
Done